Disable backward compatibility warnings in Perl module Getopt::Std
Make perl scripts in downstream_analyses executable
author: Alex Domingo (Vrije Universiteit Brussel)
diff --git a/downstream_analyses.orig/add_ka_and_ks_to_collinearity.pl b/downstream_analyses/add_ka_and_ks_to_collinearity.pl
old mode 100644
new mode 100755
index 3112029..27399af
--- a/downstream_analyses.orig/add_ka_and_ks_to_collinearity.pl
+++ b/downstream_analyses/add_ka_and_ks_to_collinearity.pl
@@ -5,6 +5,7 @@ use Bio::AlignIO;
 use Bio::Tools::Run::Alignment::Clustalw;
 use Bio::Align::DNAStatistics;
 use Getopt::Std;
+$Getopt::Std::STANDARD_HELP_VERSION=1;
 %options=();
 getopts("i:d:o:", \%options);
 if(! exists $options{i}||! exists $options{d} ||!exists $options{o})
diff --git a/downstream_analyses.orig/add_kaks_to_synteny.pl b/downstream_analyses/add_kaks_to_synteny.pl
old mode 100644
new mode 100755
index 22e5865..832f712
--- a/downstream_analyses.orig/add_kaks_to_synteny.pl
+++ b/downstream_analyses/add_kaks_to_synteny.pl
@@ -5,6 +5,7 @@ use Bio::AlignIO;
 use Bio::Tools::Run::Alignment::Clustalw;
 use Bio::Align::DNAStatistics;
 use Getopt::Std;
+$Getopt::Std::STANDARD_HELP_VERSION=1;
 %options=();
 getopts("i:d:o:", \%options);
 if(! exists $options{i}||! exists $options{d} ||!exists $options{o})
diff --git a/downstream_analyses.orig/detect_collinearity_within_gene_families.pl b/downstream_analyses/detect_collinearity_within_gene_families.pl
old mode 100644
new mode 100755
index 224c366..53f632c
--- a/downstream_analyses.orig/detect_collinearity_within_gene_families.pl
+++ b/downstream_analyses/detect_collinearity_within_gene_families.pl
@@ -1,4 +1,5 @@
 use Getopt::Std;
+$Getopt::Std::STANDARD_HELP_VERSION=1;
 %options=();
 getopts("i:d:o:", \%options);
 if(! exists $options{i}||! exists $options{d})
diff --git a/downstream_analyses.orig/detect_synteny_within_gene_families.pl b/downstream_analyses/detect_synteny_within_gene_families.pl
old mode 100644
new mode 100755
index 56c0ebf..788aeb3
--- a/downstream_analyses.orig/detect_synteny_within_gene_families.pl
+++ b/downstream_analyses/detect_synteny_within_gene_families.pl
@@ -1,4 +1,5 @@
 use Getopt::Std;
+$Getopt::Std::STANDARD_HELP_VERSION=1;
 %options=();
 getopts("i:d:o:", \%options);
 if(! exists $options{i}||! exists $options{d})
diff --git a/downstream_analyses.orig/group_collinear_genes.pl b/downstream_analyses/group_collinear_genes.pl
old mode 100644
new mode 100755
index d2a12d2..78f1cac
--- a/downstream_analyses.orig/group_collinear_genes.pl
+++ b/downstream_analyses/group_collinear_genes.pl
@@ -1,4 +1,5 @@
 use Getopt::Std;
+$Getopt::Std::STANDARD_HELP_VERSION=1;
 %options=();
 getopts("i:o:", \%options);
 if(! exists $options{i}||! exists $options{o})
diff --git a/downstream_analyses.orig/group_syntenic_genes.pl b/downstream_analyses/group_syntenic_genes.pl
old mode 100644
new mode 100755
index c0044b1..cd2030f
--- a/downstream_analyses.orig/group_syntenic_genes.pl
+++ b/downstream_analyses/group_syntenic_genes.pl
@@ -1,4 +1,5 @@
 use Getopt::Std;
+$Getopt::Std::STANDARD_HELP_VERSION=1;
 %options=();
 getopts("i:o:", \%options);
 if(! exists $options{i}||! exists $options{o})
diff --git a/downstream_analyses.orig/mkCircosInputs.pl b/downstream_analyses/mkCircosInputs.pl
old mode 100644
new mode 100755
diff --git a/downstream_analyses.orig/origin_enrichment_analysis.pl b/downstream_analyses/origin_enrichment_analysis.pl
old mode 100644
new mode 100755
index 623fe43..e356598
--- a/downstream_analyses.orig/origin_enrichment_analysis.pl
+++ b/downstream_analyses/origin_enrichment_analysis.pl
@@ -1,4 +1,5 @@
 use Getopt::Std;
+$Getopt::Std::STANDARD_HELP_VERSION=1;
 %options=();
 getopts("i:d:o:", \%options);
 if(! exists $options{i}||! exists $options{d})
